src/test/resources/focasorin/suite-desktoptest.xml

```diff
@@ -0,0 +1,15 @@
+<suite name="Desktop Test Suite">
+    <test name="Desktop Test">
```

Comment:
You're missing groups tags for include desktop here that will be run, or you can exclude mobile ones.